src/index.html

```diff
@@ -2,21 +2,128 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Moyo header</title>
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="list">
+          <li>
+            <a
+              class="link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
```

Comment:
u have enough place for it)
moved attributes to separate lines only when we have 3 or more attributes or a line longer than 80 characters.
In all the cases
```suggestion
            <a class="link" href="#">
```

Reply:
I know and didn't make separate lines. I think Prettier doing this, because I try to delete empty spase, but when I save file separate line appear again :(